http/middleware/request_id.go

```diff
@@ -0,0 +1,24 @@
+package middleware
+
+import (
+	"github.com/confetti-framework/contract/inter"
+	"github.com/google/uuid"
+)
+
+type RequestID struct{}
```

Comment:
It will be good if you add this middleware to the global middleware in: app/providers/route_service_provider.go:14

Reply:
Isn't it required to merge this pr first since I need the correct version of confetti-framework/foundation inside confetti-framework/confetti to reference to this new middleware?

Reply:
That's true. I often use replace in go.mod to edit multiple repositories at the same time. But you are absolutely right. This PR must be done first.